forte/utils/payload_factory.py

```diff
@@ -0,0 +1,159 @@
+from abc import abstractmethod
+from multiprocessing.sharedctypes import Value
+from typing import Callable
+import numpy as np
+from forte.data.ontology.top import Meta
+
+
+class PayloadFactory:
+    """
+    A class that handles the creation of payloads. It can register meta data and check its validity.
+    """
+
+    def __init__(self):
+        self.valid_meta = (
+            {}
+        )  # map from meta data name to function that handles this meta data
+
+    def register(self, meta: Meta):
```

Comment:
I don't understand what is `register` is doing, it creates a dict from meta name to an always-true boolean. What's the purpose?